app/styles/modules/_password-row.scss

```diff
@@ -68,23 +68,6 @@
 
 }
 
-/**
- * IE8 blows up when changing the type of the password field
- * to a text field. Hide the show/hide button and reset
- * the password field's padding.
- */
-.no-toggle-pw,
-.reveal-pw {
```

Comment:
The only thing that should be removed here is the `.no-toggle-pw` portion. The `.reveal-pw` portion is still used by [IE >= 10](https://github.com/mozilla/fxa-content-server/blob/8cf5c9356bc7d8735bed5c1dd851e3b5d6b65fe0/app/scripts/vendor/env-test.js#L42-L54).